packages/functions/src/randomInt.ts

```diff
@@ -0,0 +1,5 @@
+// This function *is* inclusive of both the min and max numbers
+
+export const randomInt = (min: number, max: number) => {
```

Comment:
is there really a need? feels like it is fine as is by just calling the contents of the code of this function...

Reply:
Back when I first added this to my functions, I could never remember how to do it lol

Reply:
I think shadow added it because it's include of both min and max but still just `Math.random()` and add +1 and -1 to min and max to make it inclusive.

Reply:
We can remove this imo

Reply:
I'm going to leave it because its a nice-to-have when you have to run it many times, its not used in lib anyway.